packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -125,24 +163,25 @@ export class EthereumAnchorService implements AnchorService {
    * @param params - a RequestAnchorParams object
    * @private
    */
-  private _makeAnchorRequest(params: RequestAnchorParams): Observable<AnchorServiceResponse> {
+  private _makeAnchorRequest(
+    streamID: StreamID,
+    cid: CID,
+    carFile: CAR
+  ): Observable<AnchorServiceResponse> {
     return defer(() =>
       from(
         this.sendRequest(this.requestsApiEndpoint, {
           method: 'POST',
-          body: {
-            streamId: params.streamID.toString(),
-            cid: params.tip.toString(),
-            timestamp: params.timestampISO,
-          },
+          headers: { 'Content-Type': 'multipart/form-data' },
```

Comment:
```suggestion
          headers: { 'Content-Type': 'application/vnd.ipld.car' },
```

See: https://www.iana.org/assignments/media-types/media-types.xhtml

Also, according to the spec: https://www.iana.org/assignments/media-types/application/vnd.ipld.car
Looks like we should be sending it as binary, not string.